test/utils_request.js

```diff
@@ -0,0 +1,362 @@
+import { expect } from 'chai';
+import zlib from 'zlib';
+import express from 'express';
+import { compress } from 'iltorb';
+import { requestBetter, requestLikeBrowser } from '../build/utils_request';
```

Comment:
We have a lot of `utils` files now. Perhaps we should think about a different structure. Utils really doesn't say anything about anything.

Reply:
Sure, but let's not complicate this PR any further, we can do this in another PR.